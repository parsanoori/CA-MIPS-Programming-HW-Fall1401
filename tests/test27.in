8
630434105
837467328
885914595
78005543
494213207
131725595
647422923
741660948