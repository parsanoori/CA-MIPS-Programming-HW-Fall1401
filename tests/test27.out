78005543
131725595
494213207
630434105
647422923
741660948
837467328
885914595